File: vscode/employee.js
var Employee = /** @class */ (function () {
    function Employee(empId, empName, empSalary, empMobileNo) {
        var _this = this;
        this.Display = function () {
            console.log("emp id:" + _this.id + " ", "emp name:" + _this.name + " " + "emp salary:" + _this.salary + " " + "emp mobile no:" + 7801036508);
        };
        this.id = empId;
        this.name = empName;
        this.salary = empSalary;
        this.mobileNo = empMobileNo;
    }
    return Employee;
}());
var obj = new Employee(250, "Priyanka", 23000.50, 7801036508);
obj.Display();

File: c#-practice/employee.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ConsoleApp2
{
    class employee
    {
        string empFname;
        string empLname;
        string empId;
        int empSalary;
        int BasicHRA;
        int OtherAllowences;
        int Tsalary;
        public void empDetails()
        {
            
           
            Console.WriteLine("enter the value of n:");
            int n=Convert.ToInt32(Console.ReadLine());
            Console.WriteLine("enter the values:");
            for (int i=0;i<n;i++)
            {
                Console.WriteLine("enter the emp first name");
                empFname = Console.ReadLine();
                Console.WriteLine("enter the emp last name");
                empLname = Console.ReadLine();
                Console.WriteLine("enter the emp id");
                empId = (Console.ReadLine());
                Console.WriteLine("enter the emp salary");
                empSalary = Convert.ToInt32(Console.ReadLine());
                Console.WriteLine("enter the basic HRA:");
                BasicHRA = Convert.ToInt32(Console.ReadLine());
                Console.WriteLine("enter the other allowences");
                OtherAllowences = Convert.ToInt32(Console.ReadLine());
                Console.WriteLine("the total salary is");
                Tsalary = BasicHRA + OtherAllowences;
                
                
                
         }
            for (int i = 0; i < n; i++)
            {
                Console.WriteLine(empFname);
                Console.WriteLine(empLname);
                Console.WriteLine(empId);
                Console.WriteLine(empSalary);
                Console.WriteLine(BasicHRA);
                Console.WriteLine(OtherAllowences);
                Console.WriteLine(Tsalary);
            }


        }

    }
}
